packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -261,17 +261,17 @@ const TransferPanel = (): JSX.Element => {
             return
           }
 
-          if (!bridgeTokens[selectedToken.address]?.allowed) {
```

Comment:
`bridgeTokens[selectedToken.address]?.allowed`
I don't think the allowance notion is ever useful here. should we remove it from the state altogether?